index.d.ts

```diff
@@ -469,6 +469,8 @@ export interface DailyCall {
   sendAppMessage(data: any, to?: string): DailyCall;
   addFakeParticipant(details?: { aspectRatio: number }): DailyCall;
   setShowNamesMode(mode: false | 'always' | 'never'): DailyCall;
+  showLocalVideo(bool: boolean): DailyCall;
+  showParticipantsBar(bool: boolean): DailyCall;
```

Comment:
commenting here since it's easiest to see/compare to other functions but since these are in effect setters, I think the convention here would be to call these something like:
```
setShowLocalVideo(show: boolean): DailyCall;
setShowParticipantsBar(show: boolean): DailyCall;
```
It's also odd and can be confusing to name a variable after its type like this. But that's probably the old crotchety c programmer in me coming out.

Reply:
sure, no problem!